File: app/src/main/java/com/example/ase_project_nutrismart/Database/AppDataBase.java
package com.example.ase_project_nutrismart.Database;

import androidx.annotation.NonNull;
import androidx.room.Database;
import androidx.room.DatabaseConfiguration;
import androidx.room.InvalidationTracker;
import androidx.room.RoomDatabase;
import androidx.sqlite.db.SupportSQLiteOpenHelper;

import com.example.ase_project_nutrismart.Response.LoginResponse;

@Database(entities = {LoginResponse.class},

        version = 1)
public abstract class AppDataBase extends RoomDatabase {
    public abstract UserDao userDao();

//    AppDataBase db = Room.databaseBuilder(,
//            AppDataBase.class, "database-name").build();
    @NonNull
    @Override
    protected SupportSQLiteOpenHelper createOpenHelper(DatabaseConfiguration config) {
        return null;
    }

    @NonNull
    @Override
    protected InvalidationTracker createInvalidationTracker() {
        return null;
    }

    @Override
    public void clearAllTables() {

    }
}

File: app/src/main/java/com/example/ase_project_nutrismart/Database/AppDataBase.kt
package com.example.ase_project_nutrismart.Database

import androidx.room.Database
import com.example.ase_project_nutrismart.Response.LoginResponse
import androidx.room.RoomDatabase
import com.example.ase_project_nutrismart.Database.UserDao
import androidx.room.DatabaseConfiguration
import androidx.sqlite.db.SupportSQLiteOpenHelper
import androidx.room.InvalidationTracker
import com.example.ase_project_nutrismart.Database.AppDataBase
import com.example.ase_project_nutrismart.Database.DatabaseClient
import kotlin.jvm.Synchronized
import androidx.room.Room
import androidx.room.Dao

@Database(entities = [LoginResponse::class], version = 1)
abstract class AppDataBase : RoomDatabase() {
    abstract fun userDao(): UserDao?

    //    AppDataBase db = Room.databaseBuilder(,
    //            AppDataBase.class, "database-name").build();
    override fun createOpenHelper(config: DatabaseConfiguration): SupportSQLiteOpenHelper {
        return null
    }

    override fun createInvalidationTracker(): InvalidationTracker {
        return null
    }

    override fun clearAllTables() {}
}